README.md

```diff
@@ -161,9 +161,9 @@ $ npm install npm -g
 
 <!-- links -->
 
-[dependency-status-svg]: https://david-dm.org/EatonEM/emcb-udp-master.svg
-[dependency-status-link]: https://david-dm.org/EatonEM/emcb-udp-master
-[dev-dependency-status-svg]: https://david-dm.org/EatonEM/emcb-udp-master/dev-status.svg
-[dev-dependency-status-link]: https://david-dm.org/EatonEM/emcb-udp-master#info=devDependencies
-[npm-version-svg]: https://badge.fury.io/js/emcb-udp-master.svg
-[npm-version-link]: https://badge.fury.io/js/emcb-udp-master
\ No newline at end of file
+[dependency-status-svg]: https://david-dm.org/EatonEM/emcb-udp-coordinator.svg
+[dependency-status-link]: https://david-dm.org/EatonEM/emcb-udp-coordinator
+[dev-dependency-status-svg]: https://david-dm.org/EatonEM/emcb-udp-coordinator/dev-status.svg
+[dev-dependency-status-link]: https://david-dm.org/EatonEM/emcb-udp-coordinator#info=devDependencies
```

Comment:
I don't know what these `https://david-dm.org/` links are but it seems the site no longer exists so I guess we remove these?